src/backend/distributed/commands/multi_copy.c

```diff
@@ -1177,8 +1177,9 @@ EndRemoteCopy(int64 shardId, List *connectionList, bool stopOnFailure)
 			if (stopOnFailure)
 			{
 				ereport(ERROR, (errcode(ERRCODE_IO_ERROR),
-								errmsg("failed to COPY to shard %ld on %s:%d",
-									   shardId, connection->hostname, connection->port)));
+								errmsg(
+									"failed to COPY to shard " INT64_FORMAT " on %s:%d",
+									shardId, connection->hostname, connection->port)));
```

Comment:
The extra line break is a little unfortunate :cry: Maybe do something like this?

![selection_343](https://user-images.githubusercontent.com/466333/33355009-6faea280-d46a-11e7-891e-1ff6dc6a0aa8.png)

Reply:
I had the same reaction, so :+1:

Reply:
done